var/sabath/assets/sabath/datasets/a/autoPhaseNN_aicdi.json

```diff
@@ -0,0 +1,10 @@
+{
+  "name" : "autoPhaseNN_aicdi",
```

Comment:
Grouping AutoPhaseNN data sets is informative unless they can be used independently.